test/e2e/csv_e2e_test.go

```diff
@@ -244,27 +242,22 @@ var _ = Describe("CSV", func() {
 				Scope: "Namespaced",
 			},
 		})
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 		defer cleanupCRD()
 
 		cleanupCSV, err := createCSV(GinkgoT(), c, crc, csv, testNamespace, true, false)
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 		defer cleanupCSV()
 
 		_, err = fetchCSV(GinkgoT(), crc, csv.Name, testNamespace, csvPendingChecker)
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 
 		// Shouldn't create deployment
 		_, err = c.GetDeployment(testNamespace, depName)
-		require.Error(GinkgoT(), err)
+		Expect(err).Should(HaveOccurred())
```

Comment:
This assertion is not specific enough... it will pass if (for example) the API server is temporarily unreachable, even if the deployment was created. It'll also pass if a deployment ends up being created after some time. That's what the `Consistently` Gomega assertion is for.